crates/devolutions-pedm-shared/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "devolutions-pedm-shared"
+version = "0.0.0"
+authors = ["Devolutions Inc. <infos@devolutions.net>"]
+description = "Shared code for Devolutions PEDM"
+edition = "2021"
+publish = false
+
+[dependencies]
+anyhow = { version = "1.0.86", optional = true }
+serde = { version = "1.0.203", features = ["derive"], optional = true }
+win-api-wrappers = { path = "../win-api-wrappers", optional = true }
+hyper = { version = "~0.14", features = ["client", "http1"], optional = true } # old version required for OpenAPI generator
+serde_json = { version = "1.0.117", optional = true }
+tokio = { version = "1.38.0", features = ["net", "sync"], optional = true }
+glob = { version = "0.3.1", optional = true }
+regex = { version = "1.10.4", optional = true }
+schemars = { version = "0.8.21", optional = true }
+windows-registry = { version = "0.2.0", optional = true }
+devolutions-pedm-client-http = { path = "./devolutions-pedm-client-http", optional = true }
+tower = { version = "0.3", optional = true } # old version required for OpenAPI generator
+pin-project = { version = "1.1.5", optional = true }
+
+[features]
+pedm_client = ["hyper", "serde_json", "tokio", "winapi", "policy", "serde", "devolutions-pedm-client-http", "tower", "pin-project", "anyhow"]
+policy = ["regex", "schemars", "serde", "glob"]
+build = []
```

Comment:
Ah, I am sorry, my comment was a little bit confusing, I wanted to say that instead of empty `build` feature set, there should be an empty `default` feature set, that is more in-line with rest of our code

Reply:
The `build` feature is actually for feature-gating a module called `build`, so my understanding is that we can’t replace it with `default` :thinking: